django_mongodb/fields/__init__.py

```diff
@@ -1,3 +1,4 @@
 from .auto import MongoAutoField
+from .json_field import register_fields
 
-__all__ = ["MongoAutoField"]
+__all__ = ["MongoAutoField", "register_fields"]
```

Comment:
We'll most likely be adding more fields, so in this file, use:
```python
def register_fields():
    register_jsonfield()
```
and rename `register_fields`in `json_field.py`.